x/crosschain/keeper/finalized_inbounds.go

```diff
@@ -0,0 +1,21 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+)
+
+func (k Keeper) AddFinalizedInbound(ctx sdk.Context, intxHash string, chainID int64, eventIndex uint64) {
+	finalizedInboundIndex := fmt.Sprintf("%s-%d-%d", intxHash, chainID, eventIndex)
```

Comment:
Can we create a method in `type` package to compute the key?

Reply:
9224255304d3b886d646a8ba42fc0c246655e076